### ISSUE TITLE
aria-level>6 is not output correctly for headings

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#301]
### Summary

aria-level>6 is not output correctly for headings

1. save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>heading</title>
	</head>
	<body>
		<div role=heading aria-level=0>0</div>
		<div role=heading aria-level=2>2</div>
		<div role=heading aria-level=7>7</div>
		<div role=heading>-</div>
	</body>
</html>
```
2. Read with arrow keys or navigate with H or press INS+F6

### Expected result
aria-level=7 is output as heading level 7 (see https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA12)

### Actual result 
aria-level=7 is output as heading level 2

#### JAWS version and build number

JAWS 2019.1907.42
 
#### Operating System and version

Windows 8

#### Browser and version:

Chrome 76.0.3809.100
Firefox 68.0.2
Internet Explorer 11.0.9600.19431

**Comment by sahilempire** [comment]
I want to work on this issue, please assign me.

**Comment by scottaohara** [comment]
@stevefaulkner and @BrettLewisVispero  -  with the linked `headinglevelstart` attribute that is being considered for HTML, this should be looked at / the current behavior should be reconsidered so as to not disregard heading levels beyond 6
